Add topic "localstorage" (#3933)
github/explore: topics/localstorage/index.md

---
aliases: local-storage
display_name: LocalStorage

> short_description: LocalStorage is a JavaScript API of storing arbitrary key-value data in the web browser as an alternative to cookies.

Reviewer: ```suggestion
short_description: LocalStorage is a JavaScript object that allows storing arbitrary key-value pairs in the web browser as an alternative to cookies.
```

topic: localstorage
url: https://html.spec.whatwg.org/multipage/webstorage.html#dom-localstorage-dev
wikipedia_url: https://en.wikipedia.org/wiki/Web_storage
related: sessionstorage, indexeddb, websql, web-storage, webstorage
---

> LocalStorage is a JavaScript API of storing arbitrary key-value data in the web browser as an alternative to cookies&mdash;another alterantive if the data in question goes beyond key-value pairs is to use the IndexedDB.

Reviewer: ```suggestion
LocalStorage is a JavaScript object that allows storing arbitrary key-value pairs in the web browser as an alternative to cookies&mdash;another alternative if the data in question goes beyond key-value pairs is to use the IndexedDB.
```
